Authorization/Authentication/AuthenticationStartupFilter.cs

```diff
@@ -75,5 +76,18 @@ private void RegisterHtmlPage(string uri, string content)
                     return response;
                 });
         }
+
+        private static string Fetch404Template()
+        {
+            try
+            {
+                string notFoundHTMLUrl = "/sys/error/404.html";
+                return Self.GET(notFoundHTMLUrl).Body;
+            }
+            catch
+            {
+                return "404 Page Not Found";
```

Comment:
This fallback text is not a correct HTML view for `starcounter-include`, because it is missing the `<template>` tag.

I suggest to paste here the same thing that you proposed as the content of the `/sys/error/404.html` file.

Reply:
Fixed.